protocol/x/clob/e2e/rate_limit_test.go

```diff
@@ -513,6 +712,20 @@ func TestRateLimitingShortTermOrders_GuardedAgainstReplayAttacks(t *testing.T) {
 			firstValidGTB:    &CancelOrder_Alice_Num0_Id0_Clob0_GTB20,
 			secondValidGTB:   &CancelOrder_Alice_Num1_Id0_Clob0_GTB20,
 		},
+		"Batch cancellations": {
+			blockRateLimitConfig: clobtypes.BlockRateLimitConfiguration{
+				MaxShortTermOrdersAndCancelsPerNBlocks: []clobtypes.MaxPerNBlocksRateLimit{
+					{
+						NumBlocks: 1,
+						Limit:     3, // TODO FIX THIS ONCE WEIGHTS ARE DECIDED
+					},
+				},
+			},
+			replayLessGTB:    &BatchCancel_Alice_Num0_Clob0_1_2_3_GTB5,
+			replayGreaterGTB: &BatchCancel_Alice_Num0_Clob0_1_2_3_GTB27,
+			firstValidGTB:    &BatchCancel_Alice_Num0_Clob0_1_2_3_GTB20,
+			secondValidGTB:   &BatchCancel_Alice_Num0_Clob0_1_2_3_GTB20,
```

Comment:
these are the same transactions so would get rejected even without rate limits, right?

nbd but probably better to make one of them `Alice_Num1` like the cancellation tests above

Reply:
I think you mean `BatchCancel_Alice_Num0_Clob1_1_2_3_GTB20` with CLOB1 not alice_num1 right? alice num 1 is the same subaccount owner so the rate limit is shared. I think the above test is incorrect. I will update it.

Reply:
also added some comments